lib/reporter.js

```diff
@@ -248,18 +248,32 @@ class CucumberReporter {
         return type.uri.replace(process.cwd(), '')
     }
 
-    getUniqueIdentifier (target) {
-        if (target.type === 'Hook') {
-            const name = path.basename(target.location.uri)
-            const line = target.location.line
+    getUniqueIdentifier (target, sourceLocation) {
+        let name
+        let line
 
-            return name + line
+        if (target.type === 'Hook') {
+            name = path.basename(target.location.uri)
+            line = target.location.line
+        } else if (target.type === 'ScenarioOutline') {
+            name = target.name || target.text
+            line = sourceLocation.line;
```

Comment:
Can you remove this semicolon? It's breaking the build ;-)

Reply:
I find it pretty annoying that there are no semicolons in this repo but yeah I will remove it ;) I'll pick it all up tomorrow as I got some other stuff to do today XD